src/test/java/no/unit/nva/alma/AlmaRecordParserTest.java

```diff
@@ -1,6 +1,7 @@
 package no.unit.nva.alma;
 
 import org.junit.Test;
```

Comment:
You are using two junit libraries (4 and 5). Remove Junit 4 and update the import to `import org.junit.jupiter.api.Test;
`

Reply:
Using JUnit5